environments/liveness/package.json

```diff
@@ -1,8 +1,8 @@
 {
   "private": true,
-  "name": "liveness-environment",
+  "name": "environments-in-app-messaging",
```

Comment:
I know it's draft still but reminder to self to fix name here 😀

Reply:
```suggestion
  "name": "environments-liveness",
```
Let's do it now 😈

Reply:
Let's do it now, this is me not copy pasting properly 😈 

```suggestion
  "name": "environments-liveness",
```